pkg/liqonet/tunnel/wireguard/driver.go

```diff
@@ -78,17 +83,20 @@ type ResolverFunc func(address string) (*net.IPAddr, error)
 
 // Wireguard a wrapper for the wireguard device and its configuration.
 type Wireguard struct {
-	connections map[string]*netv1alpha1.Connection
-	client      *wgctrl.Client
-	link        netlink.Link
-	conf        wgConfig
+	metrics.Metrics
+	connections                map[string]*netv1alpha1.Connection
+	connectedClusterIdentities map[string]*discv1alpha1.ClusterIdentity
```

Comment:
```suggestion
	connectedPeers map[string]*discv1alpha1.ClusterIdentity
```

Just a proposal to shorten the field name.

Reply:
I wouldn't use **connectedPeer**" to avoid it being confused with an array of wgtypes.Peer